src/ol/featureloader.js

```diff
@@ -4,6 +4,13 @@
 import {VOID} from './functions.js';
 import FormatType from './format/FormatType.js';
 
+/**
+ *
+ * @type {boolean} withCredentials Compare https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/
+ * @private
+ */
+let _withCredentials = false;
```

Comment:
Super minor, but we don't really need to prefix this with `_` (naming it `withCredentials` is fine).

Reply:
I removed the `_` and renamed the variable in the setter.